src/commands/shared.js

```diff
@@ -1,9 +1,8 @@
 const withLimit = yargs => {
-	return yargs.option('token', {
+	return yargs.option('limit', {
 		required: true,
-		type: 'string',
-		describe:
-			'GitHub personal access token. Generate one from https://github.com/settings/tokens'
+		type: 'number',
+		describe: 'limit the number of repos to return to this number'
```

Comment:
I like the old description from https://github.com/Financial-Times/ebi/pull/32/files more :)

```suggestion
		describe: 'Limit the number of repositories to search for'
```